app/src/main/java/org/mozilla/tv/firefox/utils/ServiceLocator.kt

```diff
@@ -93,7 +93,7 @@ open class ServiceLocator(val app: Application) {
     @Suppress("DEPRECATION") // We need PocketEndpointRaw until we move to a-c's impl.
     val pocketEndpointRaw by lazy { PocketEndpointRaw(appVersion, buildConfigDerivables.globalPocketVideoEndpoint) }
     val pocketVideoStore by lazy { PocketVideoStore(app, app.assets, pocketVideoJSONValidator) }
-    val pocketVideoFetchScheduler by lazy { PocketVideoFetchScheduler(isPocketEnabledByLocale) }
+    val pocketVideoFetchScheduler by lazy { PocketVideoFetchScheduler(isPocketEnabledByLocale, app.applicationContext) }
```

Comment:
nit: I think it's safe to pass in `app` as the context, as we do in other places, since the type is `Application` and not `Activity`

Reply:
This will be removed since we are deprecating pocket stuff anyway